http.go

```diff
@@ -18,14 +20,32 @@ func NewHTTPClient(ctx context.Context) *http.Client {
 	return defaultClient
 }
 
-func HTTPRequestExecutor(clientFactory transport.HTTPClientFactory) transport.HTTPRequestExecutor {
+func HTTPRequestExecutor(clientFactory transport.HTTPClientFactory, cfg *config.Backend) transport.HTTPRequestExecutor {
 	if !IsBackendEnabled() {
 		return transport.DefaultHTTPRequestExecutor(clientFactory)
 	}
 	return func(ctx context.Context, req *http.Request) (*http.Response, error) {
+		// tags for stats
+		tags := []tagGenerator{
+			func(r *http.Request) tag.Mutator { return tag.Upsert(ochttp.KeyClientHost, req.Host) },
+			func(r *http.Request) tag.Mutator { return tag.Upsert(ochttp.Host, req.Host) },
+			func(r *http.Request) tag.Mutator { return tag.Upsert(ochttp.KeyClientPath, GetAggregatedPathForBackendMetrics(cfg, req)) },
+			func(r *http.Request) tag.Mutator { return tag.Upsert(ochttp.Path, GetAggregatedPathForBackendMetrics(cfg, req)) },
+			func(r *http.Request) tag.Mutator { return tag.Upsert(ochttp.KeyClientMethod, req.Method) },
+			func(r *http.Request) tag.Mutator { return tag.Upsert(ochttp.Method, req.Method) },
+		}
+		client := clientFactory(ctx)
```

Comment:
@PhilippHeuer oh, sorry. The previous suggestion did not take this part into account. Could you remove duplicate code?